<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NixRar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="fileExplorerContainer"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Archive_Copy_As"/>
    <addaction name="actionChange_Drive"/>
    <addaction name="actionSet_Default_Password"/>
    <addaction name="separator"/>
    <addaction name="actionCopy_Files_to_Clipboard"/>
    <addaction name="actionPaste_Files_from_Clipboard"/>
    <addaction name="actionCopy_Full_Names_to_Clipboard"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionSelect_Group"/>
    <addaction name="actionDeselect_Group"/>
    <addaction name="actionInvert_Selection"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionAdd_files_to_archive"/>
    <addaction name="actionExtract_to_Specified_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionView_File"/>
    <addaction name="actionDelete_File"/>
    <addaction name="actionRename_File"/>
    <addaction name="actionPrint_File"/>
    <addaction name="separator"/>
    <addaction name="actionExtract_Without_Confirmation"/>
    <addaction name="actionProtect_Archive_from_Damage"/>
    <addaction name="actionLock_Archive"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionWizard"/>
    <addaction name="separator"/>
    <addaction name="actionScan_Archive_for_Viruses"/>
    <addaction name="actionConvert_Archive_s"/>
    <addaction name="actionRepair_Archive"/>
    <addaction name="actionConvert_Archive_to_SFX"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Files"/>
    <addaction name="actionShow_Information"/>
    <addaction name="actionGenerate_Report"/>
    <addaction name="actionBenchmark"/>
   </widget>
   <widget class="QMenu" name="menuFavourites">
    <property name="title">
     <string>Favourites</string>
    </property>
    <addaction name="actionAdd_to_Favourites"/>
    <addaction name="actionOrganize_Favourites"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuFile_List">
     <property name="title">
      <string>File List</string>
     </property>
     <addaction name="actionFlat_Folders_View"/>
     <addaction name="separator"/>
     <addaction name="actionList_View"/>
     <addaction name="actionDetails"/>
    </widget>
    <widget class="QMenu" name="menuFolder_Tree">
     <property name="title">
      <string>Folder Tree</string>
     </property>
     <addaction name="actionShow_Disk_Folders"/>
     <addaction name="actionShow_Archive_Folders"/>
    </widget>
    <widget class="QMenu" name="menuThemes">
     <property name="title">
      <string>Themes</string>
     </property>
     <addaction name="actionOrganize_Themes"/>
     <addaction name="actionGet_More_Themes"/>
     <addaction name="separator"/>
     <addaction name="actionThemeDefault"/>
    </widget>
    <widget class="QMenu" name="menuName_Encoding">
     <property name="title">
      <string>Name Encoding</string>
     </property>
     <addaction name="actionSave_Current"/>
     <addaction name="actionEncodingHelp"/>
     <addaction name="separator"/>
     <addaction name="actionEncodingDefault"/>
    </widget>
    <addaction name="actionSettings"/>
    <addaction name="actionImport_and_Export"/>
    <addaction name="actionClear_History"/>
    <addaction name="separator"/>
    <addaction name="menuFile_List"/>
    <addaction name="menuFolder_Tree"/>
    <addaction name="menuThemes"/>
    <addaction name="menuName_Encoding"/>
    <addaction name="separator"/>
    <addaction name="actionView_Log"/>
    <addaction name="actionClear_Log"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_Topics"/>
    <addaction name="actionNixRAR_GithubPage"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommands"/>
   <addaction name="menuTools"/>
   <addaction name="menuFavourites"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFile"/>
   <addaction name="actionExtractTo"/>
   <addaction name="actionTest"/>
   <addaction name="actionView"/>
   <addaction name="actionDelete"/>
   <addaction name="actionFind"/>
   <addaction name="actionWizard"/>
   <addaction name="actionInfo"/>
   <addaction name="separator"/>
   <addaction name="actionRepair"/>
  </widget>
  <widget class="QToolBar" name="navigationBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionAddFile">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/rar-format.png</normaloff>:/Icons/icons/rar-format.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add File(s) to Archive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExtractTo">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/folders.png</normaloff>:/Icons/icons/folders.png</iconset>
   </property>
   <property name="text">
    <string>Extract To</string>
   </property>
   <property name="toolTip">
    <string>Extract files to directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionTest">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/checklist.png</normaloff>:/Icons/icons/checklist.png</iconset>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <property name="toolTip">
    <string>Test Archive</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionView">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/open-book.png</normaloff>:/Icons/icons/open-book.png</iconset>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>View archive info</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/bin.png</normaloff>:/Icons/icons/bin.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete item from archive</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/loupe.png</normaloff>:/Icons/icons/loupe.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>Find item in archive</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionWizard">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/magic-wand.png</normaloff>:/Icons/icons/magic-wand.png</iconset>
   </property>
   <property name="text">
    <string>Wizard</string>
   </property>
   <property name="toolTip">
    <string>Start the archive wizard</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/info.png</normaloff>:/Icons/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="toolTip">
    <string>Show archive information</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRepair">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/band-aid.png</normaloff>:/Icons/icons/band-aid.png</iconset>
   </property>
   <property name="text">
    <string>Repair</string>
   </property>
   <property name="toolTip">
    <string>Repair archive</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/open.png</normaloff>:/Icons/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Archive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/exit.png</normaloff>:/Icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="NixRar.qrc">
     <normaloff>:/Icons/icons/info.png</normaloff>:/Icons/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_Archive_Copy_As">
   <property name="text">
    <string>Save Archive Copy As</string>
   </property>
  </action>
  <action name="actionChange_Drive">
   <property name="text">
    <string>Change Drive</string>
   </property>
  </action>
  <action name="actionSet_Default_Password">
   <property name="text">
    <string>Set Default Password</string>
   </property>
  </action>
  <action name="actionCopy_Files_to_Clipboard">
   <property name="text">
    <string>Copy Files to Clipboard</string>
   </property>
  </action>
  <action name="actionPaste_Files_from_Clipboard">
   <property name="text">
    <string>Paste Files from Clipboard</string>
   </property>
  </action>
  <action name="actionCopy_Full_Names_to_Clipboard">
   <property name="text">
    <string>Copy Full Names to Clipboard</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionSelect_Group">
   <property name="text">
    <string>Select Group</string>
   </property>
  </action>
  <action name="actionDeselect_Group">
   <property name="text">
    <string>Deselect Group</string>
   </property>
  </action>
  <action name="actionInvert_Selection">
   <property name="text">
    <string>Invert Selection</string>
   </property>
  </action>
  <action name="actionExtract_to_Specified_Folder">
   <property name="text">
    <string>Extract to Specified Folder</string>
   </property>
  </action>
  <action name="actionView_File">
   <property name="text">
    <string>View File</string>
   </property>
  </action>
  <action name="actionDelete_File">
   <property name="text">
    <string>Delete File</string>
   </property>
  </action>
  <action name="actionRename_File">
   <property name="text">
    <string>Rename File</string>
   </property>
  </action>
  <action name="actionPrint_File">
   <property name="text">
    <string>Print File</string>
   </property>
  </action>
  <action name="actionExtract_Without_Confirmation">
   <property name="text">
    <string>Extract Without Confirmation </string>
   </property>
  </action>
  <action name="actionProtect_Archive_from_Damage">
   <property name="text">
    <string>Protect Archive from Damage</string>
   </property>
  </action>
  <action name="actionLock_Archive">
   <property name="text">
    <string>Lock Archive</string>
   </property>
  </action>
  <action name="actionScan_Archive_for_Viruses">
   <property name="text">
    <string>Scan Archive for Viruses</string>
   </property>
  </action>
  <action name="actionConvert_Archive_s">
   <property name="text">
    <string>Convert Archive(s)</string>
   </property>
  </action>
  <action name="actionRepair_Archive">
   <property name="text">
    <string>Repair Archive</string>
   </property>
  </action>
  <action name="actionConvert_Archive_to_SFX">
   <property name="text">
    <string>Convert Archive to SFX</string>
   </property>
  </action>
  <action name="actionFind_Files">
   <property name="text">
    <string>Find Files</string>
   </property>
  </action>
  <action name="actionShow_Information">
   <property name="text">
    <string>Show Information</string>
   </property>
  </action>
  <action name="actionGenerate_Report">
   <property name="text">
    <string>Generate Report</string>
   </property>
  </action>
  <action name="actionBenchmark">
   <property name="text">
    <string>Benchmark</string>
   </property>
  </action>
  <action name="actionAdd_files_to_archive">
   <property name="text">
    <string>Add File(s) to Archive</string>
   </property>
  </action>
  <action name="actionAdd_to_Favourites">
   <property name="text">
    <string>Add to Favourites</string>
   </property>
  </action>
  <action name="actionOrganize_Favourites">
   <property name="text">
    <string>Organize Favourites</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionImport_and_Export">
   <property name="text">
    <string>Import and Export</string>
   </property>
  </action>
  <action name="actionClear_History">
   <property name="text">
    <string>Clear History</string>
   </property>
  </action>
  <action name="actionFlat_Folders_View">
   <property name="text">
    <string>Flat Folders View</string>
   </property>
  </action>
  <action name="actionList_View">
   <property name="text">
    <string>List View</string>
   </property>
  </action>
  <action name="actionDetails">
   <property name="text">
    <string>Details</string>
   </property>
  </action>
  <action name="actionShow_Disk_Folders">
   <property name="text">
    <string>Show Disk Folders</string>
   </property>
  </action>
  <action name="actionShow_Archive_Folders">
   <property name="text">
    <string>Show Archive Folders</string>
   </property>
  </action>
  <action name="actionOrganize_Themes">
   <property name="text">
    <string>Organize Themes</string>
   </property>
  </action>
  <action name="actionGet_More_Themes">
   <property name="text">
    <string>Get More Themes</string>
   </property>
  </action>
  <action name="actionThemeDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionSave_Current">
   <property name="text">
    <string>Save Current</string>
   </property>
  </action>
  <action name="actionEncodingHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionEncodingDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionView_Log">
   <property name="text">
    <string>View Log</string>
   </property>
  </action>
  <action name="actionClear_Log">
   <property name="text">
    <string>Clear Log</string>
   </property>
  </action>
  <action name="actionHelp_Topics">
   <property name="text">
    <string>Help Topics</string>
   </property>
  </action>
  <action name="actionNixRAR_GithubPage">
   <property name="text">
    <string>NixRAR GitHub Page</string>
   </property>
   <property name="toolTip">
    <string>NixRAR GitHub Page</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="NixRar.qrc"/>
 </resources>
 <connections/>
</ui>
